.github/workflows/CI.yml

```diff
@@ -16,13 +16,14 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
+      max-parallel: 2
```

Comment:
I will try to increase the max-parallel value.

Reply:
I have increased max-parallel to 4. It takes around 30 minutes to complete all workflows.

Reply:
This configuration has been removed!